src/translations/en.json

```diff
@@ -1933,15 +1933,19 @@
           "users_privileges_note": "The user group feature is a work in progress. The user will be unable to administer the instance via the UI. We're still auditing all management API endpoints to ensure that they correctly limit access to administrators.",
           "picker": {
             "headers": {
-              "name": "Name",
+              "name": "Display name",
+              "username": "User name",
               "group": "Group",
-              "system": "System"
+              "system": "System generated",
+              "is_active": "Active",
+              "is_owner": "Owner"
             },
             "add_user": "Add user"
           },
           "editor": {
             "caption": "View user",
-            "name": "Name",
+            "name": "Display Name",
+            "username": "User Name",
```

Comment:
We should not duplicate these translations?

Reply:
Removed the duplication between "add_user" and "editor", but left it for the table headers since there we are usually only capitalizing the first letter, where as for input fields in dialogs we capitalize all.